core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -87,6 +87,9 @@ angular.module('oppia').directive('storyNodeEditor', [
             $scope.nodeIdToTitleMap =
               $scope.story.getStoryContents().getNodeIdsToTitleMap(
                 $scope.storyNodeIds);
+            $scope.nodeIdToDescriptionMap =
```

Comment:
Use (..) and break after ( here.

Reply:
I had removed it as requested by different comment.